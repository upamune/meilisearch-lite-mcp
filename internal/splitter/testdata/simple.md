# Header 1

This is the first paragraph.

## Header 2

- List item 1
- List item 2

```go
func main() {
	fmt.Println("Hello")
}
```

Another paragraph.
