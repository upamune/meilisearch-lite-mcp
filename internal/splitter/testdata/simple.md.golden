[
  {
    "Text": "This is the first paragraph.",
    "StartIdx": 12,
    "EndIdx": 40,
    "Type": "text",
    "Headings": [
      "Header 1"
    ]
  },
  {
    "Text": "List item 1\n- List item 2",
    "StartIdx": 57,
    "EndIdx": 82,
    "Type": "text",
    "Headings": [
      "Header 1",
      "Header 2"
    ]
  },
  {
    "Text": "```go\nfunc main() {\n\tfmt.Println(\"Hello\")\n}\n```",
    "StartIdx": 90,
    "EndIdx": 128,
    "Type": "code",
    "Headings": [
      "Header 1",
      "Header 2"
    ]
  },
  {
    "Text": "Another paragraph.",
    "StartIdx": 133,
    "EndIdx": 151,
    "Type": "text",
    "Headings": [
      "Header 1",
      "Header 2"
    ]
  }
]
